dubbo-config/dubbo-config-spring/src/main/java/com/alibaba/dubbo/config/spring/beans/factory/annotation/ReferenceAnnotationBeanPostProcessor.java

```diff
@@ -360,11 +400,17 @@ private Object buildReferenceBean(Reference reference, Class<?> referenceClass)
      * @param beanClass {@link Class}
      * @return
      */
-    private static String generateReferenceBeanCacheKey(Reference reference, Class<?> beanClass) {
+    private String generateReferenceBeanCacheKey(Reference reference, Class<?> beanClass) {
 
         String interfaceName = resolveInterfaceName(reference, beanClass);
 
-        String key = reference.group() + "/" + interfaceName + ":" + reference.version();
+        String key = reference.url() + "/" + interfaceName +
```

Comment:
using StringBuilder maybe more nice .

Reply:
It's the same in JVM, switch to StringBuilder is welcome